exe/wallet/Main.hs

```diff
@@ -182,9 +189,10 @@ main = do
                                   Logging
 -------------------------------------------------------------------------------}
 
-initTracer :: IO (Trace IO Text)
-initTracer = do
+initTracer :: Severity -> IO (Trace IO Text)
+initTracer minSeverity = do
     c <- defaultConfigStdout
+    setMinSeverity c minSeverity
     tr <- setupTrace (Right c) "simple"
     appendName "text" tr
```

Comment:
Not related to this PR but we should review the `"simple"` and `"text"` too here, since logged currently gets displayed as: `[iohk.simple.text] <log message>` ... Not sure we need to append anything, and `simple` could just be `cardano-wallet` I believe :smile:

Reply:
> I wonder if and when we could have a configuration file at some point. I admit I don't like the idea too much and would prefer having everything in CLI and/or in the ENV.

The number of options is certainly becoming longer, and it certainly makes the help text unwieldy to look at. Right now, we display **all options** for **all commands** in the help text.

Perhaps we could alleviate this by using a different CLI parsing framework that has a nicer way to display the various options?

With a different framework, we could display a general overview when the user types `--help`, and then give greater levels of detail when the user asks for help in a specific area (for example `cardano-wallet mnemonic --help`) or for a specific command, like `mnemonic generate --help`.

Reply:
Yes! And that is already planned :) Most likely coming for our next sprint.

Reply:
> Not related to this PR but we should review the `"simple"` and `"text"` too here, since logged currently gets displayed as: `[iohk.simple.text] <log message>` ... Not sure we need to append anything, and `simple` could just be `cardano-wallet` I believe

Great point! I've added a task to do this.